docs/release-notes/8.8.asciidoc

```diff
@@ -0,0 +1,97 @@
+* Allows you to specify how to handle alert suppression for alerts with missing fields ({pull}155055[#155055]).
+* Updates the rule settings Actions interface ({pull}154526[#154526]).
+* Adds a warning message to tell you when the maximum number of alerts for a single rule execution is exceeded ({pull}154112[#154112]).
+* Updates how browser field descriptions are provided to {kib} ({pull}153498[#153498]).
+* Makes alert links on the Entity Analytics page navigate to the Alerts page instead of opening in Timeline ({pull}153372[#153372]).
+* Enables multi-level grouping for alerts on the Alerts page, based on various fields ({pull}152862[#152862]).
+* Makes it possible to link directly to a version of the Alerts page with filters active, and makes widgets on the Detection and Response dashboard do so ({pull}152714[#152714]).
+* Updates the visualizations throughout {elastic-sec} to Lens visualizations ({pull}150531[#150531]).
+* Adds a "Share alert" button to the alert details flyout, and makes alert flyout URLs more sharable ({pull}148800[#148800]).
+* Adds a note to the Rules page when a maintenance window is running ({pull}155386[#155386]).
+* Adds a global search bar to the Detections and Response and Entity Analytics dashboards ({pull}156832[#156832]).
+
+
+[discrete]
+[[bug-fixes-8.8.0]]
+==== Bug fixes
+
+* Fixes a bug that interfered with the default time range when you opened an alert in Timeline ({pull}156884[#156884]).
+* Updates where the technical preview tags appear for host risk score features ({pull}156659[#156659], {pull}156514[#156514]).
+* Fixes a bug that could cause the alerts page to become unresponsive after you entered an invalid query ({pull}156542[#156542]).
+* Updates the colors used for entity analytics graphs to match the colors used for alerts graphs ({pull}156383[#156383]).
+* Fixes a bug that caused errors on the Data Quality dashboard when a `basePath` was configured ({pull}156233[#156233]).
+* Fixes a bug that could cause problems when different users edited a single timeline simultaneously ({pull}155663[#155663]).
+* Fixes a bug that could cause the wrong number of rules to appear in the interface for duplicating rules ({pull}155959[#155959]).
+* Fixes a bug that could cause a blank option to appear in the Create rule exception interface ({pull}155221[#155221]).
+* Fixes issues that affected tags in the Add rule exception component of the Shared Exception Lists page ({pull}155219[#155219]).
+* Fixes a bug that displayed an outdated count of affected rules on the Shared Exception Lists page ({pull}155108[#155108]).
+* Improves performance for rendering indicator match alerts on the Alerts page ({pull}154821[#154821]).
+* Fixes a bug that could affect alert prevalence counts on the Alerts page ({pull}154544[#154544]).
+* Adds the "Add to timeline" action to alert counts on the Detections and Response dashboard and the Entity Analytics dashboard ({pull}154299[#154299]).
```

Comment:
@stephmilovic this seems more like an enhancement than a fixed bug because the change happened because the alert count redirect link was updated to the Alerts page. Should we move this summary to the enhancement section?

```suggestion
* Adds the *Add to timeline* inline action to alert counts on the Detections and Response dashboard and the Entity Analytics dashboard ({pull}154299[#154299]).
```

Reply:
Also I think the option is now "Investigate in timeline" (jump to Timeline UI and populate query there), not "Add to timeline" (add field to Timeline, but don't jump there).